submission/submission-handlers/src/main/kotlin/ac/uk/ebi/biostd/handlers/config/ApplicationConfig.kt

```diff
@@ -49,7 +53,11 @@ class Services {
     fun bioStudiesWebConsumer(
         restTemplate: RestTemplate,
         extSerializationService: ExtSerializationService
-    ): BioStudiesWebConsumer = BioStudiesWebConsumer(restTemplate, extSerializationService)
+    ): BioStudiesWebConsumer =
+        BioStudiesWebConsumer(
+            restTemplate.apply { messageConverters = getConverters(messageConverters) },
```

Comment:
is there any integration test at least we can add as a TODO to verify this? (that utf8 strings are correctly parser) otherwise it looks to me that we can easy change bean configuration